tests/kernel/install_ltp.pm

```diff
@@ -126,25 +145,26 @@ sub install_from_git {
     }
     assert_script_run("git clone -q --depth 1 $url" . $tag, timeout => 360);
     assert_script_run 'cd ltp';
+    # It is a shallow clone so 'git describe' won't work
+    script_run 'git log -1 --pretty=format:"git-%h" > /opt/ltp_version';
     assert_script_run 'make autotools';
     assert_script_run("$configure $extra_flags", timeout => 300);
     assert_script_run 'make -j$(getconf _NPROCESSORS_ONLN)', timeout => 1440;
     script_run 'export CREATE_ENTRIES=1';
     assert_script_run 'make install', timeout => 360;
-    assert_script_run "find ~/ltp/testcases/open_posix_testsuite/conformance/interfaces -name '*.run-test' > ~/openposix_test_list.txt";
-    # It is a shallow clone so 'git describe' won't work
-    script_run 'git log -1 --pretty=format:"git %h" > /opt/ltp_version';
+    assert_script_run "find ~/ltp/testcases/open_posix_testsuite/conformance/interfaces -name '*.run-test' > ~/openposix-test-list-git";
 }
 
 sub install_from_repo {
     zypper_call('in qa_test_ltp', dumb_term => 1);
-    assert_script_run q(find ${LTPROOT:-/opt/ltp}/testcases/bin/openposix/conformance/interfaces/ -name '*.run-test' > ~/openposix_test_list.txt);
+    assert_script_run q(find ${LTPROOT:-/opt/ltp}/testcases/bin/openposix/conformance/interfaces/ -name '*.run-test' > ~/openposix-test-list-pkg);
     script_run 'rpm -q qa_test_ltp > /opt/ltp_version';
 }
 
```

Comment:
@richiejp I'd split changes into more commits (where possible and not cause pain), so the patch is more readable. Like this renaming of output file.